LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -77,6 +77,13 @@ public ILabAccount getLabAccount(@NonNull final LabQuery labQuery) throws LabApi
         return getLabAccountObject(configInfo);
     }
 
+    public ILabAccount getLabAccount(@NonNull final String upn) throws LabApiException {
```

Comment:
I suggest we also add the method to `ILabClient` and then add the `overrides` annotation here